### ISSUE TITLE
Tortoise needs more testing by not me. I don't have the hardware to run it alongside any of the models I have. As per discussion in #885, this has 3 Tortoise implementations: official, fast, and MRQ

### ISSUE THREAD

**Issue by 6561368163** [oobabooga/text-generation-webui#3277]
Tortoise needs more testing by not me. I don't have the hardware to run it alongside any of the models I have. As per discussion in #885, this has 3 Tortoise implementations: official, fast, and MRQ

_Originally posted by @da3dsoul in https://github.com/oobabooga/text-generation-webui/issues/1106#issuecomment-1509468898_

**Comment by 78Alpha** [comment]
Might be easier to run normal text to speech and run it through RVC to get the voice of your choice. I have used tortoise a lot and it can climb to 10 GB VRAM all on its own. RVC is about 1 GB and can alter an hour of audio in a couple minutes.

Of the tortoise implementations, MrQ is certainly the fastest. about 30 seconds for a sentence.

**Comment by da3dsoul** [comment]
Lol no fast is the fastest, at almost realtime on a 3070

**Comment by da3dsoul** [comment]
Fair. I've been busy. If someone wants to try to update and improve my PR, go for it. I might get to it eventually

**Comment by 78Alpha** [comment]
At this point,  tortoise was superseded by XTTS. 

And there's now LLM TTS models that should lead without too much extra tinkering. Training them is a nightmare but possible.

**Comment by da3dsoul** [comment]
I agree. It was a cool experiment. There are clearly better options now
